abhishekb91/petstore-openapi3
# Added Basic Project Structure

### WHAT
Setting up the project with a basic API structure

### WHY
So that we can build on top of it 

### HOW
* Added server stub from oapi-codegen
* Compiled and added swagger-ui to the project using statik
* Created a basic echo project

### TEST
* Ensure the API up and running using the README
* Ensure API document page is accessible when the project is running
